framework/db/mysql/QueryBuilder.php

```
@@ -399,6 +409,28 @@ public function selectExists($rawSql)
+     *
+     * @return array{string, string|null} Column definition and CHECK clause
+     */
+    private function extractCheckFromColumnDefinition($definition)
+    {
+        $checkRegex = '/CHECK *(\(([^()]|(?-2))*\))/i';
+
+        if (preg_match($checkRegex, $definition, $checkMatches) !== 1) {
+            return [$definition, null];
+        }
+
+        $definitionWithoutCheck = str_replace($checkMatches[0], '', $definition);
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $definitionWithoutCheck. Keep variable name length under 20. (undefined)

(LongVariable)